.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,13 +11,25 @@
 
 ## Checklist
 
-<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--
+
+  Put an `x` in the completed tasks.
+
+  If a task is not relevant, `x` it, then strike through the text e.g.:
+  - [x] ~~This task is not relevant.~~
+
+-->
+
+Tasks for all changes:
+
+- [ ] 1. I have linked this PR to its Jira ticket.
+- [ ] 2. I have run git pre-commits.
+- [ ] 3. I have added and/or updated relevant tests.
+- [ ] 4. I have updated relevant documentation.
+- [ ] 5. I have considered the cross-team impact (and have PR approval from both Core & Demographics if necessary).
+- [ ] 6. I have successfully deployed this change to a sandbox and witnessed unit and e2e tests passing.
 
-- [ ] I have followed the code style of the project.
-- [ ] I have added tests to cover my changes.
-- [ ] I have updated the documentation accordingly.
-- [ ] I have considered the cross-team impact when creating this PR and where possible written tests to validate for all affected.
-- [ ] I have built a sandbox with code from this PR and ensured the unit tests and end-to-end tests have run successfully.
-- [ ] A member of all teams has approved this PR if there is a major code change. (If this is a minor change and you are confident this is not necessary then you may accept the standard approval process of 2 approvals from any teams.)
+Additional tasks for UI changes (delete if not applicable):
 
----
+- [ ] 1. I have run the [`SANDBOX - UI Smoke Test`](https://github.com/NHSDigital/national-document-repository/actions/workflows/ui-smoketest.yml) workflow against the deployed sandbox and witnessed it passing.
```

Comment:
If I'm going to be really pedantic if there is a link to this workflow should there not be a link to the deploy-sandbox.yml workflow?